pkg/alexa/gen/model_test.go

```diff
@@ -97,6 +98,24 @@ func TestModelBuilder_Build(t *testing.T) {
 }
 
 // individual functions
+func TestModelWith(t *testing.T) {
+	mb := gen.NewModelBuilder().
+		WithLocaleRegistry(registry).
+		WithInvocation("MyInvocationKey").
+		AddLocale("en-US", "invoke me")
+	// fails because "en-US" is already in registry!
+	assert.Nil(t, mb)
+
+	mb = gen.NewModelBuilder().
```

Comment:
You have 2 test cases in a single test. Split these and name them by their expectation.